tools/lint/main_ni.c

```diff
@@ -287,6 +287,9 @@ help(int shortout)
             "                create an exact YANG schema context. If specified, the '-F'\n"
             "                parameter (enabled features) is ignored.\n\n");
 
+    printf("  -X, --enable-deref-within-leafref\n"
```

Comment:
Again, sorry if I was not clear but this now seems too specific to me. To be future proof, I would reuse the flag name, something like `--extended-leafref`.

Reply:
Done